.github/workflows/commit-checker.yml

```diff
@@ -0,0 +1,42 @@
+name: Commit checker
+
+on:
+  pull_request:
+
+jobs:
+  commit-checker:
+    name: Commit checker
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 3
```

Comment:
Lets put the initial depth on 4. That is the merge commit, the master, and 2 commits in the PR. Average PR has one commit, so that means we capture just more than average.